CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+This guide summarises the [Github guide](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll) for adding new content to a Jekyll site. The "publishing source" is the `master` branch; anything added to `master` will be published automatically. Firstly, please create an issue and branch for the content you wish to add.
```

Comment:
I think we should expand on "create an issue and branch".

Perhaps, "create an issue which explains briefly what you are planning to add (and can be used for discussion)".

Do we want people to work on forks?  I think probably not, in which case should we say something about how to request access?

Reply:
Sure. How about using the issue for discussion of the topic and ideas, then the PR for discussion of the actual content?

I'm ashamed to say I don't know what we should do about people requesting access. Is there any way you can do this through github, or is it a case of them sending me an email and then me adding them? This is my first time dealing with an "organisation" account too.

Reply:
I've just clarified the contributing guidance; I hope that's about right now.

It looks like users need to be added to an organisation; they can't request access through github. Perhaps we should provide a contact email for this. Personal or UKAEA?

Reply:
Use UKAEA email.